frontend/src/components/GGenericInputFields.vue

```diff
@@ -0,0 +1,243 @@
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
```

Comment:
introduce GGenericInputField.vue component that handles one field for better readability. Not sure if a component like GGenericInputFields.vue is then still necessary